protocol/contracts/beanstalk/sun/SeasonFacet/Weather.sol

```diff
@@ -70,65 +99,87 @@ contract Weather is Sun {
     
         Decimal.D256 memory deltaPodDemand;
 
-        // If Sow'd all Soil
+        // `s.w.thisSowTime` is set to the number of seconds in it took for 
+        // Soil to sell out during the current Season. If Soil didn't sell out,
+        // it remains `type(uint32).max`.
         if (s.w.thisSowTime < type(uint32).max) {
             if (
                 s.w.lastSowTime == type(uint32).max || // Didn't Sow all last Season
-                s.w.thisSowTime < SOWTIMEDEMAND || // Sow'd all instantly this Season
+                s.w.thisSowTime < SOW_TIME_DEMAND_INCR || // Sow'd all instantly this Season
                 (s.w.lastSowTime > C.getSteadySowTime() &&
                     s.w.thisSowTime < s.w.lastSowTime.sub(C.getSteadySowTime())) // Sow'd all faster
-            ) deltaPodDemand = Decimal.from(1e18);
-            else if (
+            ) {
+                deltaPodDemand = Decimal.from(1e18);
+            } else if (
                 s.w.thisSowTime <= s.w.lastSowTime.add(C.getSteadySowTime())
-            )
+            ) {
                 // Sow'd all in same time
                 deltaPodDemand = Decimal.one();
-            else deltaPodDemand = Decimal.zero();
```

Comment:
Only `deltaPodDemand` wrapped in brakcets, the following two `s.w...` lines are excluded